public/cover.css

```diff
@@ -0,0 +1,176 @@
+/*
+ * Globals
+ */
+
+/* Links */
+a,
+a:focus,
+a:hover {
+  color: #fff;
+}
+.nav .open>a, .nav .open>a:focus, .nav .open>a:hover {
```

Comment:
Be consistent with your multiple selectors: either put them all on a line (my personal preference), or each on a line, but be consistent :wink: